FWCore/Utilities/interface/EDMException.h

```diff
@@ -16,8 +16,7 @@ namespace edm {
   namespace errors {
 
     // If you add a new entry to the set of values, make sure to
-    // update the translation map in EDMException.cc, the actions
-    // table in FWCore/Framework/src/Actions.cc, and the configuration
```

Comment:
@Dr15Jones I did not see `Actions.cc` anywhere so I took the liberty of removing it from the comment. Is there any other place (in addition to `cmsExceptionsFatalOption_cff`) that would need to be updated?

Reply:
That appears to be it to me.